conjurapi/resource.go

```diff
@@ -25,3 +32,55 @@ func (c *Client) CheckPermission(resourceID, privilege string) (bool, error) {
 		return false, fmt.Errorf("Permission check failed with HTTP status %d", resp.StatusCode)
 	}
 }
+
+// Resource fetches a single user-visible resource by id.
+func (c *Client) Resource(resourceID string) (resource map[string]interface{}, err error) {
+	req, err := c.router.ResourceRequest(resourceID)
+	if err != nil {
+		return
+	}
+
+	resp, err := c.SubmitRequest(req)
+	if err != nil {
+		return
+	}
+
+	data, err := response.DataResponse(resp)
+	if err != nil {
+		return
+	}
+
+	resource = make(map[string]interface{})
+	if err = json.Unmarshal([]byte(data), &resource); err != nil {
+		return
+	}
+
+	return resource, err
```

Comment:
Seems superfluous wit the `if` above. This should be functionally equivalent:
```
	resource = make(map[string]interface{})
	err = json.Unmarshal([]byte(data), &resource)
 	return resource, err
```

Reply:
Yup, you're right.